### ISSUE TITLE
Editing in the CLI

### ISSUE THREAD

**Issue by BeckyK102125** [github/copilot-cli#1573]
### Describe the bug

I am blind and am using the CLI version of Copilot. I am using it with Windows 11 and the screen reader Non-Visual Desktop Access (NVDA). NVDA is updated to its latest version. When editing what I am writing in the CLI when I move through the letters or words the screen reader is reading the status bar instead of the  information I have written. 

### Steps to reproduce the behavior

1. Turn on screen reader
2. Open the CLI
3. Begin writing a prompt
4. Ensure the prompt is two lines
5. use the left, right, up and down arrow keys if the prompt is two or more lines. If the prompt is one line use left or right. You should hear the status bar being read and/or space, space space. It does seem to alternated 
I have reproduced it several times where it says the status bar and many times where is says space space sapce. 

### Expected behavior

If the left arrow is pressed you should hear the previous character spoken, if the right arrow is pressed the next character is spoken. If the prompt is multi line, if you press the down arrow the next line should be spoken of your prompt, if the up arrow is pressed the previous line should be spoken.

**Comment by taylorarndt** [comment]
Also having this issue.
